src/app/Plugins/Timer/Components/timer/timer.component.ts

```diff
@@ -0,0 +1,137 @@
+import { Component, OnInit } from '@angular/core';
+import { MatToolbarModule } from '@angular/material/toolbar';
+import { TimerDataService } from '../../Services/timer-data.service';
+import { Timer } from '../../Objects/timer';
+import { LocalDateTime } from '../../../../Objects/LocalDateTime';
+
+/*
+  Component for the timer-plugin. It recieves data from the backend with a custom timer-dataservices and displays the data over
+  the html-template.
+*/
+@Component({
+  selector: 'app-timer',
+  templateUrl: './timer.component.html',
+  styleUrls: ['./timer.component.css']
+})
+
+export class TimerComponent implements OnInit {
+
+  timerData: Timer[];
+  oneTimerData: Timer;
+  selectedTimer: Timer;
+  date: Date;
+  hour: number;
+  minute: number;
+  second: number;
+  /*
+    Providing the data-service for the clock-component
+  */
+  constructor(private readonly clockService: TimerDataService) { }
+
+  ngOnInit() {
+    this.timerData = new Array<Timer>();
+    this.getTimers();
+    setInterval(() => this.date = new Date(), 1000);
```

Comment:
Yes it is used to refresh the displayed time each second, because it is an easy way to change values each second. --> html has to refresh and shows the actual remaining time